Regression/WarpX-tests.ini

```
@@ -1189,3 +1189,17 @@ numthreads = 2
 compileTest = 0
 doVis = 0
 compareParticles = 0
+
+[initial_distribution]
+buildDir = .
+inputFile = Examples/Tests/initial_distribution/inputs
+dim = 3
+restartTest = 0
+useMPI = 1
+numprocs = 1
+useOMP = 1
+numthreads = 1
```

```suggestion
useMPI = 1
numprocs = 2
useOMP = 1
numthreads = 2
```
I think it is safer if we always test with a parallel run.